src/cephadm/cephadm

```diff
@@ -903,13 +903,9 @@ class CephIscsi(object):
 
     def get_tcmu_runner_container(self):
         # type: () -> CephContainer
-        tcmu_container = get_container(self.ctx, self.fsid, self.daemon_type, self.daemon_id)
+        tcmu_container = get_container(self.ctx, self.fsid, self.daemon_type, self.daemon_id + '.tcmu')
         tcmu_container.entrypoint = '/usr/bin/tcmu-runner'
         tcmu_container.cname = self.get_container_name(desc='tcmu')
-        # remove extra container args for tcmu container.
-        # extra args could cause issue with forking service type
-        tcmu_container.container_args = []
-        set_pids_limit_unlimited(self.ctx, tcmu_container.container_args)
```

Comment:
Please update `qa/workunits/cephadm/test_iscsi_pids_limit.sh` that was added in #44579 to also test for `cat /sys/fs/cgroup/pids/pids.max` being `max` in both containers, before spawning sleeps.